backend/sublet/views.py

```diff
@@ -130,12 +166,55 @@ def list(self, request, *args, **kwargs):
             queryset = queryset.filter(beds=beds)
         if baths:
             queryset = queryset.filter(baths=baths)
-
         # Serialize and return the queryset
-        serializer = SimpleSubletSerializer(queryset, many=True)
+        serializer = SubletSerializerSimple(queryset, many=True)
         return Response(serializer.data)
 
 
+class CreateImages(generics.CreateAPIView):
+    serializer_class = SubletImageSerializer
+    http_method_names = ["post"]
+    permission_classes = [SubletImageOwnerPermission | IsSuperUser]
+    parser_classes = (
+        MultiPartParser,
+        FormParser,
+    )
+
+    def get_queryset(self, *args, **kwargs):
+        sublet = get_object_or_404(Sublet, id=int(self.kwargs["sublet_id"]))
+        return SubletImage.objects.filter(sublet=sublet)
+
+    # takes an image multipart form data and creates a new image object
+    def post(self, request, *args, **kwargs):
+        images = request.data.getlist("images")
+        sublet_id = int(self.kwargs["sublet_id"])
+        self.get_queryset()  # check if sublet exists
+        img_serializers = []
+        for img in images:
+            img_serializer = self.get_serializer(data={"sublet": sublet_id, "image": img})
+            img_serializer.is_valid(raise_exception=True)
+            img_serializers.append(img_serializer)
+        instances = [img_serializer.save() for img_serializer in img_serializers]
+        data = [SubletImageURLSerializer(instance=instance).data for instance in instances]
+        return Response(data, status=status.HTTP_201_CREATED)
+
+
+class DeleteImage(generics.DestroyAPIView):
+    serializer_class = SubletImageSerializer
+    http_method_names = ["delete"]
+    permission_classes = [SubletImageOwnerPermission | IsSuperUser]
+    queryset = SubletImage.objects.all()
+
+    def destroy(self, request, *args, **kwargs):
+        queryset = self.get_queryset()
```

Comment:
`nit`: no need to extract to queryset variable since only used once and relatively simple lol